packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -172,6 +177,15 @@ class _DropdownMenuItemButtonState<T> extends State<_DropdownMenuItemButton<T>>
       padding: widget.padding,
       child: widget.route.items[widget.itemIndex],
     );
+    final BorderRadius itemBorderRadius;
+    if (widget.borderRadius == null)
+      itemBorderRadius = BorderRadius.zero;
+    else if (widget.itemIndex == 0)
+      itemBorderRadius = BorderRadius.only(topLeft: widget.borderRadius!.topLeft, topRight: widget.borderRadius!.topRight);
+    else if (widget.itemIndex == widget.route.items.length - 1)
+      itemBorderRadius = BorderRadius.only(bottomLeft: widget.borderRadius!.bottomLeft, bottomRight: widget.borderRadius!.bottomRight);
+    else
+      itemBorderRadius = BorderRadius.zero;
```

Comment:
someone else will probably review this, and I might be wrong, but I think Flutter team prefers to use { } instead of inlining everything. And this is a long if/else chain.

Reply:
I'm not quite sure I understand. Could you please elaborate on what you mean by 'use { }'?

Reply:
if (a) {b} else {c} instead of if (a) b else c

Reply:
Yes. I think it's still optional, but most Flutter developers use braces for if/else expressions. Just a little safer in the long run.